test/src/unit-enumerations.cc

```diff
@@ -144,6 +145,7 @@ struct EnumerationFx {
   Config cfg_;
   Context ctx_;
   EncryptionKey enc_key_;
+  shared_ptr<MemoryTracker> memory_tracker_;
```

Comment:
Took me a second before I realized this isn't parented off the ArraySchema PR so we had to add it in both PRs. We'll just need to rebase this after that other PR is merged.